graphics/fonts/bdf.cpp

```diff
@@ -700,6 +700,15 @@ BdfFont *BdfFont::loadFromCache(Common::SeekableReadStream &stream) {
 	return new BdfFont(data, DisposeAfterUse::YES);
 }
 
+static char *new_strdup(const char *in) {
```

Comment:
I recommend to rename it to scumm_strdup(), and move to common/str.h, similar to scumm_stricmp() and scumm_scrnicmp()